### PR TITLE
RGRIDT-894: Fixing GetRowData

### PR DESCRIPTION
**Pull request by glundgren93**
This PR fixes GetRowData when Grid is Single Entity.

### What was happening
* GetRowData was not working properly on grids with single entity.

### What was done
* Created method that check is grid is single entity or not and returns dataItem from row.

### Checklist
* [x] tested locally
* [x] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)
* [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]
👍

File: code/src/WijmoProvider/Features/Rows.ts [hunk 1]
```diff
@@ -266,6 +267,16 @@ namespace WijmoProvider.Feature {
             ) as CssClassInfo;
         }
 
+        public getRowData(rowNumber: number): string {
```
Comment (tmbp95) [inline code/src/WijmoProvider/Features/Rows.ts:270]: I would make it different. This method might be useful for other use cases and in my opinion, it should not return a stringified JSON but instead the dataItem. Then, in the GridAPI you can use the stringify whenever you need it to be a string.

Comment (glundgren93) [reply code/src/WijmoProvider/Features/Rows.ts:270]: makes sense.

Comment (glundgren93) [reply code/src/WijmoProvider/Features/Rows.ts:270]: Changed it

**Review by tmbp95 — APPROVED** [review]

**Review by tmbp95 — DISMISSED** [review]

**Review by tmbp95 — APPROVED** [review]

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

**Review by rugoncalves — APPROVED** [review]

**Pull request merged**